src/test/java/uk/gov/gchq/koryphe/impl/binaryoperator/DeduplicateConcatTest.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright 2017 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.gov.gchq.koryphe.impl.binaryoperator;
+
+import org.junit.Test;
+import uk.gov.gchq.koryphe.binaryoperator.BinaryOperatorTest;
+import uk.gov.gchq.koryphe.util.JsonSerialiser;
+import java.io.IOException;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+public class DeduplicateConcatTest extends BinaryOperatorTest {
+
+    @Test
+    public void shouldRemoveDuplicate() {
```

Comment:
Please could you add some more tests to check for things like:
null
"a,b,"
",a,b"
""